app/controllers/case_workers/claims_controller.rb

```diff
@@ -5,9 +5,9 @@ class CaseWorkers::ClaimsController < CaseWorkers::ApplicationController
   def index
     @claims = case tab
       when 'current'
-        current_user.claims_to_manage.submitted
+        current_user.rolable.claims.submitted
```

Comment:
The claims relationship is on Advocate/CaseWorker and not on the user model. The relationship for advocates to claims is one to many, many to many for case workers to claims.

Could delegate claims to persona by putting in user model:

delegate :claims, to: :persona

Then we could all current_user.claims